trunner/target/emulated.py

```diff
@@ -94,3 +94,15 @@ def __init__(self):
     @classmethod
     def from_context(cls, _: TestContext):
         return cls()
+
+
+class ARMv8R52_MPS3_AN536QemuTarget(QemuTarget):
```

Comment:
From what I can see, we don't use `_` in other target class names. I would stick to one consistent naming convention.

Reply:
@adamdebek please apply the suggestion